src/track/trackref.h

```diff
@@ -110,6 +110,10 @@ bool operator!=(const TrackRef& lhs, const TrackRef& rhs) {
     return !(lhs == rhs);
 }
 
+inline uint qHash(const TrackRef& key, uint seed) {
```

Comment:
These extensions could be extracted into a separate PR to reduce the size of this PR.

Reply:
Now done: #2761.